gz: increase timeout for service request

dakejahl opened the pull request:
Closes #23705
Closes #23809

leechor commented:
> Closes #23705 Closes #23809

it works for me, by the way, line 915 maybe modified too.

Jaeyoung-Lim reviewed (changes requested):
This is called on a loop. Wouldn't it make sense to modify the user message that it is retrying, rather than increase the timeout?

dakejahl commented:
> This is called on a loop. Wouldn't it make sense to modify the user message that it is retrying, rather than increase the timeout?

I think not having the warning in the first place is optimal. I also updated the text and changed from err to warn

Jaeyoung-Lim reviewed (approved):
Thanks!
